DQMServices/Core/interface/DQMEDHarvester.h

```diff
@@ -24,13 +26,14 @@ class DQMEDHarvester
   // implicit assignment operator
   // implicit destructor
   virtual void beginRun(edm::Run const&, edm::EventSetup const&) {};
-  virtual void analyze(edm::Event const&, edm::EventSetup const&) final {};
+  virtual void produce(edm::Event&, edm::EventSetup const&) override final {};
```

Comment:
@ghellwig I added override to beginRun and endRun. This just moves the warning to the headers that declare beginRun and ednRun for classes inheriting from DQMEDHarvester.